### PR TITLE
docs: translate migration docs

### PR DESCRIPTION
**Pull request by Timeless0911**
## Summary

translate migration docs

image

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: website/docs/zh/guide/migration/modernjs-module.mdx [hunk 1]
```diff
@@ -1 +1,186 @@
...
+    {
+      bundle: false,
+      dts: true,
+      format: 'esm',
+    },
+  ],
+});
+```
+
+## Typescript 类型定义
```
Comment (fi3ework) [inline website/docs/zh/guide/migration/modernjs-module.mdx:90]:
```suggestion
## TypeScript 类型定义
```

File: website/docs/en/guide/migration/modernjs-module.mdx [hunk 2]
```diff
@@ -81,9 +87,9 @@ export default defineConfig({
 });
 ```
 
-## Typescript
+## Typescript Declaration
```
Comment (fi3ework) [inline website/docs/en/guide/migration/modernjs-module.mdx:90]:
```suggestion
## TypeScript Declaration
```

File: website/docs/zh/guide/basic/output-format.mdx [hunk 3]
```diff
@@ -45,7 +45,7 @@ Rslib 支持四种 JavaScript 文件的输出格式：`esm`、`cjs`、`umd` 和
   - 一些 Node.js 工具和库仍然对 ESM 有限制或不支持，需要解决方法或额外配置。
   - 你必须在导入路径中显式包含文件扩展名，这可能会很麻烦，尤其是在使用 TypeScript 或其他转译语言时。
 
-#### 发布 [ESM 和 CJS（双重）](https://antfu.me/posts/publish-esm-and-cjs#compatibility) 包
+#### 发布 [ESM 和 CJS (双重)](https://antfu.me/posts/publish-esm-and-cjs#compatibility) 包
```
Comment (fi3ework) [inline website/docs/zh/guide/basic/output-format.mdx:48]: Full-width symbols should be used in Chinese.

Comment (Timeless0911) [reply website/docs/zh/guide/basic/output-format.mdx:48]: image

The Chinese brackets are a bit wide and don't look pretty in the title.

**Review by fi3ework — APPROVED** [review]

File: website/docs/zh/guide/basic/output-format.mdx (hunk 3 above)
Comment (fi3ework) [reply website/docs/zh/guide/basic/output-format.mdx]: Since this is not a mistake, and it makes sense, I am open to it.

**Pull request merged**